js/privatebin.js

```diff
@@ -2295,6 +2129,192 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
             I18n._(me.$attachmentLink, label, me.$attachmentLink.attr('download'));
         }
 
+
+        /*
+         * Read file data as dataURL using the FileReader API
+         * https://developer.mozilla.org/en-US/docs/Web/API/FileReader#readAsDataURL()
+         */
+        me.readFileData = function (file) {
+            if (typeof FileReader === 'undefined') {
+                // revert loading status…
+                AttachmentViewer.hideAttachment();
+                AttachmentViewer.hideAttachmentPreview();
+                Alert.showError(I18n._('Your browser does not support uploading encrypted files. Please use a newer browser.'));
```

Comment:
Please remove the `I18n.`. The new version of this function automatically translates the text.